components/tools/OmeroJava/test/utests/gateway/model/PolylineAndPolygonTest.java

```diff
@@ -283,24 +255,6 @@ public void testSavePointsLegacyModeForPolyline() {
             assertEquals(p.getX(), p1.getX());
             assertEquals(p.getY(), p1.getY());
         }
```

Comment:
we should still keep the tests for the legacy mode. We still need to be sure we can read such shapes